tests/test_composition.py

```diff
@@ -647,13 +693,21 @@ def test_instancing(self):
             sq[1:] = [it, copy.deepcopy(it)]
         self.assertEqual(len(sq), 2)
 
+    def test_delete_parent_container(self):
+        # deleting the parent container should null out the parent pointer
+        it = otio.core.Item()
+        sq = otio.schema.Track(children=[it])
+        del sq
+        self.assertIsNone(it.parent())
+
     def test_range(self):
         length = otio.opentime.RationalTime(5, 1)
         tr = otio.opentime.TimeRange(otio.opentime.RationalTime(), length)
         it = otio.core.Item(source_range=tr)
         sq = otio.schema.Track(children=[it])
         self.assertEqual(sq.range_of_child_at_index(0), tr)
 
+        del sq
```

Comment:
This feels a little odd. Maybe the test should use sq, sq2, etc. instead of reusing the same variable?
Or just putting a comment explaining why you need to `del sq` would help someone reading this & wondering why you need to do that.